packages/core/src/tree.js

```diff
@@ -94,20 +95,24 @@ const Tree = props => {
         inputIconRenderer={inputIconRenderer}
         clearIconRenderer={clearIconRenderer}
       />
-      <Items styles={styles} getStyles={getStyles} height={itemsHeight}>
-        {leaves &&
-          leaves.length > 0 &&
-          leaves.map(item => (
-            <Item
-              getStyles={getStyles}
-              searchTerm={searchTerm}
-              item={item}
-              onClick={onClick}
-              forwardIconRenderer={forwardIconRenderer}
-              selectedItem={selectedItem}
-            />
-          ))}
-      </Items>
+      {customItemsRenderer ? (
+        React.cloneElement(customItemsRenderer, { leaves, searchTerm, onClick })
+      ) : (
+        <Items styles={styles} getStyles={getStyles} height={itemsHeight}>
+          {leaves &&
+            leaves.length > 0 &&
+            leaves.map(item => (
+              <Item
+                getStyles={getStyles}
+                searchTerm={searchTerm}
+                item={item}
+                onClick={onClick}
+                forwardIconRenderer={forwardIconRenderer}
+                selectedItem={selectedItem}
```

Comment:
You don't want to pass these two?

Reply:
as I see selected Item we pass externally

Reply:
ForwardIconRenderer the same

Reply:
I think it's important we maintain the same API.
IT would be very confusing if the icons work in one instance but if you pass a custom list they don't.

Reply:
Just a ping on this so that we can merge

Reply:
Updated